### PR TITLE
Replaced html table 

### PR DESCRIPTION
**Pull request by maridoroshuk**
#### Summary

Replaced html table with markdown and deleted Description section for better look.
What do you think? Does this look better? 

#### Related issues

Issue #15743 
#### Metadata

- [ ] Adds a new document
- [x] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/html/element/input/index.md [hunk 1]
```diff
@@ -27,297 +27,32 @@ How an `<input>` works varies considerably depending on the value of its [`type`
...
-        {{EmbedLiveSample("exampledatetime",200,75,"","", "nobutton")}}
-      </td>
-    </tr>
-  </tbody>
-</table>
+| Type                                                        | Description                                                                                                                                                                                                                                                                                       |
+|-------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| {{HTMLElement("input/button",   "button")}}                 | A push button with no default behavior   displaying the value of the value attribute, empty by default.                                                                                                                                                                                           |
```
Comment (dipikabh) [inline files/en-us/web/html/element/input/index.md:32]:
```suggestion
| {{HTMLElement("input/button",   "button")}}                 | A push button with no default behavior displaying the value of the [value](/en-US/docs/Web/HTML/Element/input#value) attribute, empty by default.                                                                                                                                                                                           |
```

Similarly, there were links in other input types that will need to be put back - file, image, radio, and range

File: files/en-us/web/html/element/input/index.md [hunk 2]
```diff
@@ -27,297 +27,32 @@ How an `<input>` works varies considerably depending on the value of its [`type`
...
+| {{HTMLElement("input/url",   "url")}}                       | A field for entering a URL. Looks like a text input, but has validation   parameters and relevant keyboard in supporting browsers and devices with   dynamic keyboards.                                                                                                                           |
+| {{HTMLElement("input/week",   "week")}}                     | A control for entering a date consisting of a   week-year number and a week number with no time zone.                                                                                                                                                                                             |
+| **Obsolete values**                                         |                                                                                                                                                                                                                                                                                                   |
```
Comment (dipikabh) [inline files/en-us/web/html/element/input/index.md:54]:
```suggestion
| **Obsolete value**                                         |                                                                                                                                                                                                                                                                                                   |
```

**Review by dipikabh — CHANGES_REQUESTED** [review]

**Comment by maridoroshuk** [comment]
@dipikabh Hey! sorry for this mess, I added all links I've missed. All is fine now :)

File: files/en-us/web/html/element/input/index.md [hunk 3]
```diff
@@ -27,297 +27,32 @@ How an `<input>` works varies considerably depending on the value of its [`type`
...
+| {{HTMLElement("input/hidden",   "hidden")}}                 | A control that is not displayed but whose value   is submitted to the server. There is an example in the next column, but it's   hidden!                                                                                                                                                          |
+| {{HTMLElement("input/image",   "image")}}                   | A graphical submit button. Displays an image defined by the src attribute. The [`alt`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#alt) attribute displays if the   image [src](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#src) is   missing                                                                                                                                                                                                                                                                                                                                                           |
```
Comment (OnkarRuikar) [inline files/en-us/web/html/element/input/index.md:40]:
```suggestion
| {{HTMLElement("input/image",   "image")}}                   | A graphical submit button. Displays an image defined by the src attribute. The [`alt`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#alt) attribute displays if the image [`src`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#src) is missing                                                                                                                                                                                                                                                                                                                                                           |
```

File: files/en-us/web/html/element/input/index.md [hunk 4]
```diff
@@ -27,297 +27,32 @@ How an `<input>` works varies considerably depending on the value of its [`type`
...
+| {{HTMLElement("input/week",   "week")}}                     | A control for entering a date consisting of a   week-year number and a week number with no time zone.                                                                                                                                                                                             |
+| **Obsolete values**                                         |                                                                                                                                                                                                                                                                                                   |
+| {{HTMLElement("input/datetime",   "datetime")}}             | {{deprecated_inline}} A control for entering a   date and time (hour, minute, second, and fraction of a second) based on UTC   time zone.                                                                                                                                                         |
```
Comment (OnkarRuikar) [inline files/en-us/web/html/element/input/index.md:55]:
```suggestion
| {{HTMLElement("input/datetime",   "datetime")}} {{deprecated_inline}} | A control for entering a   date and time (hour, minute, second, and fraction of a second) based on UTC   time zone.                                                                                                                                                         |
```
The badge needs to be near type.

**Review by OnkarRuikar — COMMENTED** [review]
It would be great if we get rid of the two extra spaces between words e.g. " to be...selected", "opening a...color",  "and...day",  "with no...time zone".

**Review by wbamberg — COMMENTED** [review]
@dipikabh , @maridoroshuk , what do you think about making this into a Markdown `<dl>` instead of a table? It would be more consistent with our other pages that way. And Markdown tables with long rows make for quite unreadable source (we usually resort to HTML tables https://developer.mozilla.org/en-US/docs/MDN/Contribute/Markdown_in_MDN#when_to_use_html_tables when the rows get too long).

**Review by wbamberg — COMMENTED** [review]
Sorry if I wasn't clear. I meant a Markdown `<dl>`, using the syntax described in https://developer.mozilla.org/en-US/docs/MDN/Contribute/Markdown_in_MDN#definition_lists 

Only we prefer dashes to asterisks:

```
- item1
  - : description1
- item2
  - : description2
```

...then you can also use Markdown instead of HTML in the items.

Also I suggest just dropping `datetime` at the end.

Also it looks like you are using Prettier to autoformat files. I like Prettier and would love to be running it on all our stuff, but it makes it harder to review PRs when there are lots of extra changes to look at, so could you revert those changes?

**Comment by maridoroshuk** [comment]
@wbamberg Yes, got you!

**Pull request closed without merging**